### DISCUSSION TITLE
Have nuxt in a separate package

### DISCUSSION THREAD

**Discussion by mariusa** [cdmoro/bootstrap-vue-3#840] [Ideas]
Hi,

What do you think about having`nuxt` support in a separate package, please? 

Not everybody is using it and the current install brings a lot deps due to `nuxt`.

Thanks

**Comment by VividLemon** [comment]
This wont be able to be fully implemented until https://github.com/bootstrap-vue/bootstrap-vue-next/projects/5 is finished

**Reply by mariusa** [reply]
Thanks for considering!

**Reply by mariusa** [reply]
Thanks again! With 0.8 release, with nuxt in a separate package, my package-lock decreased from 228k to 177k :tada:
